### PR TITLE
Last checked on date

### PR DESCRIPTION
**Pull request by mirhamasala**
- Add `checked_on` key to YAML
- Create a function to get the value of the `checked_on` key that is used in the template

### CODE REVIEW

**Comment by mirhamasala** [comment]
Reminder: Update @gabriel-seow & @jannwong once this is PR is merged

File: src/constants/travel.ts [hunk 1]
```diff
@@ -39,3 +39,7 @@ export function findSourcesForCountry(countryCode: string) {
 export function findTravelIdeasForCountry(countryCode: string) {
   return Object.values(countries[countryCode]?.travel_ideas || []);
 }
+
+export function findLastCheckedOnDate() {
```
Comment (yurikoval) [inline src/constants/travel.ts:43]: I don't think we need to export this as a function. This value is fixed, so we can export the value itself.

File: src/constants/travel.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@ import TravelDefaults from '@/constants/travel.yaml';
 
 export const Travel = TravelDefaults;
 
+export const sourcesLastCheckedOn = Travel.checked_on;
+
 const { countries } = Travel;
```
Comment (yurikoval) [inline src/constants/travel.ts:10]: can be combined: `const { countries, checked_on } = Travel;`

File: src/constants/travel.ts [hunk 3]
```diff
@@ -5,8 +5,9 @@ import TravelDefaults from '@/constants/travel.yaml';
 
 export const Travel = TravelDefaults;
 
-const { countries } = Travel;
+const { countries, checkedOn } = Travel;
 
+export default checkedOn;
```
Comment (yurikoval) [inline src/constants/travel.ts:10]: can't export this as default

**Pull request merged**